docs/user/user_doc_index.rst

```diff
@@ -15,3 +15,4 @@ End-user CloudFerry documentation
     glance_image_migration
     cinder_migration
     copy_mechanisms
+    estimate_migration
```

Comment:
I think that logically we should place `estimate_migration` above than now - probably after `installation` or after `configuration`. Because this step should occur before actually migration process. What do you think?